types/pnicorp/placePod/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "PlacePod (Surface & In-Ground Mount)",
+  "name": "PlacePod",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed